pho/src/components/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { FaCompress, FaUsers, FaVolumeMute, FaVolumeUp } from 'react-icons/fa';
+
+import './Toolbar.css';
+
+const Toolbar: React.FC<{
+  numberOfParticipants: number;
+  setIsMuted: Function;
+  isMuted: boolean;
+  visible: boolean;
+}> = ({ numberOfParticipants, setIsMuted, isMuted, visible }) => {
+  const [isFullscreen, setIsFullscreen] = useState(false);
+  return (
+    <div
+      className="toolbar-container"
+      style={visible ? {} : { display: 'none' }}
+    >
+      <div className="toobar-item">
+        {numberOfParticipants}
+        <FaUsers style={{ paddingLeft: '5px' }} />
+      </div>
+      <div className="toobar-item">
+        {isMuted ? (
+          <FaVolumeMute onClick={() => setIsMuted()} />
+        ) : (
+          <FaVolumeUp onClick={() => setIsMuted()} />
+        )}
+      </div>
+      <div className="toobar-item">
+        <FaCompress
+          onClick={() => {
+            if (isFullscreen) {
+              document.exitFullscreen().catch(console.log);
+            } else {
+              const element = document.getElementById('root');
+
```

Comment:
I think we could use `document.body` here to fix the drawer issue in fullscreen mode. The drawer element stays outside of `<div id='root'>`.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/115213709/199513037-ce65b8d2-ce64-4cc8-9f31-244c1e1d65d8.png">